boot/zephyr/Kconfig.serial_recovery

```diff
@@ -202,4 +202,11 @@ config BOOT_SERIAL_IMG_GRP_IMAGE_STATE
 	  If y, image states will be included with image lists and the set
 	  state command can be used to mark an image as test/confirmed.
 
+config BOOT_SERIAL_IMG_GRP_SLOT_INFO
+	bool "Slot info"
+	default y
```

Comment:
I am not sure we should bring it by default from the beginning; this increases flash and and surprised developers will be forced to check what happened and disable the feature.

Reply:
What about default if updateable images > 1? The problem that always comes up is (with multi image devices) knowing what "slot" to use to upload to, on single image devices yes this feature is mostly useless (maybe max image size is useful) but for multi image devices especially those that support uploading via MCUboot and application, it gives clarity over where to upload

Reply:
Have moved it to be > 1 image. Though had to add a fix for the app size not being populated (it is populated if you enter serial recovery by way of there being no application which I was testing before, but not through button press). Unfortunately because you now need to duplicate the sector data, it has significantly bumped RAM.
Without:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       43188 B        64 KB     65.90%
             RAM:       28448 B       448 KB      6.20%
```
With:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       43912 B        64 KB     67.00%
             RAM:       32560 B       448 KB      7.10%
```
+724 bytes flash, +4112 bytes RAM. Will ask David if we can just make it static in the file or add a define for it because that is pretty ridiculous